SPEC.md

```diff
@@ -216,3 +229,6 @@ IPAM plugin examples:
 #### Notes
  - Routes are expected to be added with a 0 metric.
  - A default route may be specified via "0.0.0.0/0". Since another network might have already configured the default route, the CNI plugin should be prepared to skip over its default route definition.
+
+## Well-known Error Codes
+- `1` - Incompatabile CNI version
```

Comment:
That's what I'm saying here (https://github.com/appc/cni/pull/80/files#diff-916b3e1e005fd96e3a0546715235477dR65) just with not so many words. Having backwards compat within same major version is part of SemVer spec.